### PR TITLE
Merge notebooks, skafos scripts and new docs into Skafos repo

### PR DESCRIPTION
**Pull request by griffinwalkerMM**

### CODE REVIEW

**Comment by griffinwalkerMM** [comment]
@mfriedel for some reason unable to add you as a reviewer so adding a comment. Here is proposed classifier repo.

Happy to make any changes you think are necessary/ easier for the audience receiving this.

**Comment by griffinwalkerMM** [comment]
I don't have write access as griffinwalkerMM, I'll use this same process for the Recommender example but after that I will use the skafos account directly.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,73 @@
...
+![alt text](https://cdn-images-1.medium.com/max/1120/1*vUB5BP_Fs09Add2P5QV16Q.png "Logo Title Text 1")  |  ![alt_text](https://cdn-images-1.medium.com/max/1120/1*ZXXlUT8jOuYJw6mpGCJEJQ.png "Logo Title Text 2")
+ 
+Thanks to [Viktor Malyi's 4-part article Run or Walk](https://towardsdatascience.com/run-or-walk-detecting-user-activity-with-machine-learning-and-core-ml-part-1-9658c0dcdd90) for the images.
+  
+## What can you find in this repo?
+- `activity_classifer.py` - The main executable that trains the activitiy classifier.
+- `activity.model` & `activity.mlmodel` - A pre-trained TuriCreate model and CoreML model.
+- `activity_classifier.ipynb` - A Jupyter Notebook containing the same code as 'activity_classifier'
+- `Activity_Data_Integration_Example.ipynb` - A Jupyter Notebook containing an example of how to get your own data working in the TuriCreate framework.
+- `/models` - a directory that contains a saved TuriCreate model (`.model`) and a saved CoreML model (`.mlmodel`)
```
Comment (mfriedel) [inline README.md]: ## What can you find in this repo? 

- `activity_classifer.py` - The main executable that trains the activitiy classifier.
- `activity_classifier.ipynb` - A Jupyter Notebook containing the same code as 'activity_classifier'
- `Activity_Data_Integration_Example.ipynb` - A Jupyter Notebook containing an example of how to get your own data working in the TuriCreate framework.
- `/models` - a directory that contains a saved TuriCreate model (`.model`) and a saved CoreML model (`.mlmodel`) These models can be used as is, or you can regenerate them via the tutorial below. 
- `/common` - a director that contains functions and code called from the main activity classifier script.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,73 @@
...
-TODOs: 
 
-* Break into multiple files
-* Skafosify it
-* Generate or re-partition so we have new data for scoring.  
+## What do I do with it?
+- If you want to use this model in your app to detect activities such as standing, walking, running, standing up, sitting down and laying down, download the model and incorporate it into your app.
+- If you want to train the model yourself, sign up for a Skafos project, follow these steps and watch it train.
+- If you want to train the model on different data, take a look below.
+
```
Comment (mfriedel) [inline README.md]: ## What do I do with it? --> Change this section to three parts that we can expand upon later. Outline is below. 

## How to use this repo: 

### 1. Use the `.mlmodel` file in your app to classify activities. 

### 2. Use the provided code and example data to train an activity classifier


### 3. Train an activity classifier on different data

File: README.md [hunk 3]
```diff
@@ -1,9 +1,73 @@
...
+
+## The Data
+The data used to tran this model looks like the following:
```
Comment (mfriedel) [inline README.md]: _This is an outline of what I think this section should look like. The comments apply to the entire section._

## Input Data

The data used to train this model is the publicly available HAPT data set. (Provide link) When using the `activity_classifier.py` to train the model, we use a copy of this data set that has been placed on a public S3 bucket. 

In order to train this model with Turi Create, the HAPT Data set needed to be wrangled into the format shown below. The code to do this was adapted from Turi Create's example [LINK] and can be found here [LINK]

_Here show the table with Experiment, Activity, etc_

The general format of this data is that each experiment contains 6 measurements. 3 axes collected from an accelerometer and 3 axes collected from a gyroscope. Each experiment also contains a labeled activity (e.g. walking, running, standing, etc.)

In general, so long as you can collect the 3 dimensions from each sensor and associate them with an activity and the "experiment" of that activity, you can pre-train the model on any activity you would like (jumping, throwing, falling). An example of how to do this is described above. 

Keep the three links at the end about how the data was generated.

File: README.md [hunk 4]
```diff
@@ -1,9 +1,73 @@
...
+| 748402 |           61 | climbing_upstairs |          0.834722 |         -0.358333 |        -0.0986111 |       1.17714 |      1.02381  |    -0.388816  |
+| 748403 |           61 | climbing_upstairs |          0.802778 |         -0.329167 |        -0.104167  |       1.21348 |      0.91813  |    -0.332311  |
+| 748404 |           61 | climbing_upstairs |          0.770833 |         -0.2875   |        -0.0986111 |       1.32619 |      0.846659 |    -0.202502  |
+| 748405 |           61 | climbing_upstairs |          0.718056 |         -0.268056 |        -0.0555556 |       1.45875 |      0.783129 |    -0.0736093 |
+
+- To find out more about how this data was generated, take a look at the following [link](https://archive.ics.uci.edu/ml/datasets/Human+Activity+Recognition+Using+Smartphones).
+- To see the data being generated, take a look at this [video](https://www.youtube.com/watch?v=XOEN9W05_4A).
+- If your app doesn't already collect motion data, check out [this app](https://itunes.apple.com/us/app/sensor-kinetics/id579040333?mt=8) that allows you to mess around with the gyroscope and accelerometer on your iPhone.
+
+## How do I integrate my own data?
```
Comment (mfriedel) [inline README.md]: ## How do I integrate my own data? 

I think we can remove this section, and link the notebook to item #3 above.

File: README.md [hunk 5]
```diff
@@ -1,9 +1,73 @@
...
+- Turi Create provides some example code on how to aggregate iOS sensor readings and how to make predictions [here](https://apple.github.io/turicreate/docs/userguide/activity_classifier/export_coreml.html)
+- In Viktor Malyi's article mentioned above, he uploaded his own running and walking data to Kaggle. 
+- To see how we took Viktor's data and formatted it such that TuriCreate could train a model over his data, check out this repo's [Activity_Data_Integration_Example.ipynb](https://github.com/griffinwalkerMM/TuriActivityClassifier/blob/master/Activity_Data_Integration_Example.ipynb) notebook.
+
+## Going beyond the TuriCreate example
```
Comment (mfriedel) [inline README.md]: ## Going beyond the TuriCreate example --> Remove this section

File: README.md [hunk 6]
```diff
@@ -1,9 +1,73 @@
...
+Out of the box, Turi Create gives a great,easy implementation of an Activity Classifier. To go beyond this and make your classifier more sophisticated, Turi Create has some [really great documentation](https://apple.github.io/turicreate/docs/userguide/activity_classifier/).
+
+- For example, both Viktor Malyi's data and the HAPT data set in the original example sampled data from a given experiment/session at a rate of 50 Hz. The example used a `prediction_window` of 50. What does this mean?
+
+*" ... if we want to produce a prediction every 5 seconds, and the sensors are sampled at 50Hz - we would set the prediction_window to 250 (5 sec * 50 samples per second)... Since we have created the model with samples taken at 50Hz and set the prediction_window to 50, we will get one prediction per second. "*.
+
+One can play around with the prediction window to see how it changes model output but also it may have to be adjusted depending on the frequency at which you are sampling your activity data. 
+
+### To try this out:
```
Comment (mfriedel) [inline README.md]: To try this out --> Move this section to the "how to use this code to train your own model." section above.

**Pull request merged**